samples/features/sql-big-data-cluster/deployment/private-bdc/README.md

```diff
@@ -0,0 +1,51 @@
+# Deploy BDC in private AKS cluster with User-defined Route (UDR)
+
+This repository contains the scripts that you can use to deploy a private BDC cluster in Azure Kubernetes Service (AKS) with advanced networking ( CNI ). 
+
+This repository contains 3 bash scripts : 
+- deploy-private-aks.sh : You can use it to deploy private AKS cluster with private endpoint, it fits the use case that you need to deploy BDC with a private endpoint with AKS private cluster. 
+- deploy-private-aks-udr.sh : You can use it to deploy private AKS cluster with private endpoint, it fits the use case that you need to deploy BDC with a private endpoint with AKS private cluster and limit egress traffic with UDR ( User-defined Routes ). 
+- deploy-private-bdc.sh : You can use it to deploy Big Data Clusters ( BDC ) in private deployment mode on private AKS cluster with or without User-defined routes based on your project requirements. 
+
+
+## Prerequisite
+
+The following table lists common big data cluster tools and how to install them:
```

Comment:
Please keep just the pre-requisites *required* to run the scripts. You can point to the documentation for other useful tools for BDC. Otherwise it will be difficult to maintain these lists in multiple places.